### ISSUE TITLE
Actions are 900ms in grunge stack compared to 30ms in indie 

### ISSUE THREAD

**Issue by soricinae** [remix-run/grunge-stack#122]
### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

1. Create a new grunge stack and run it
2. Create account
3. Create 2 notes
4. Open network developer tool in browser
5. Click between the two notes repeatedly and watch the network call durations

### Expected Behavior

Loading notes takes 30ms (like it does in Indie stack)

### Actual Behavior

Loading notes takes 800-900ms

**Comment by davidfilat** [comment]
Yes, I have the same issue. It takes about 100-300 miliseconds with the indie stack and 1.6 seconds on AWS Lambda (1.1 seconds with ARC sandbox) with the grunge stack.
